internal/service/clusteroutagesimulation/data_source_cluster_outage_simulation_test.go

```diff
@@ -1,162 +0,0 @@
-package clusteroutagesimulation_test
-
-import (
```

Comment:
I like this approach of combining the data tests with the resource tests. The extra terraform code is worth reducing the number of tests and duplication 👏

Reply:
i think so, i'll discuss in the tech meeting